crates/nostr-mls-memory-storage/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nostr-mls-memory-storage"
+version = "0.41.0"
+edition = "2021"
+description = "In-memory database for nostr-mls that implements the NostrMlsStorageProvider Trait"
+authors = ["Jeff Gardner <j@jeffg.me>", "Yuki Kishimoto <yukikishimoto@protonmail.com>", "Rust Nostr Developers"]
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+readme = "README.md"
+rust-version.workspace = true
+keywords = ["nostr", "mls", "openmls", "memory"]
+
+[dependencies]
+lru.workspace = true
+nostr = { workspace = true, features = ["std"] }
+nostr-mls-storage.workspace = true
+openmls_memory_storage = { version = "0.3", default-features = false }
+parking_lot = { version = "0.12", features = ["send_guard"] }
```

Comment:
I guess the `send_guard` feature here can be disabled, since the guards are never sent between threads